app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,10 @@
+<h1> Contestants Index Page</h1>
+
+<% @contestants.each do |contestant| %>
+  <section id="contestant-<%= contestant.id %>">
+    <%= contestant.name %>
+    <% contestant.projects.each do |project| %>
+      <%= project.name%>
+    <% end %>
```

Comment:
You can do a nested iteration here. You could have also created a method in your `Contestant` model that plucks al the project names. Something like:
```ruby
def projects_list
  projects.pluck(:name).to_sentence
end
```
Then instead of doing additional iterating on line 6 you could just call `contestant.projects_list`.